views/insight_monitors.view.lkml

```diff
@@ -126,14 +126,25 @@ view: insight_monitors {
 
   dimension: size_monitor {
     type: yesno
-    sql: ${TABLE}."SIZE_MONITOR" ;;
+    # sql: ${TABLE}."SIZE_MONITOR" ;;
+    sql: ${volume_change_monitor} or ${unchanged_size_monitor} ;;
```

Comment:
It's unrelated to the prefix change right?

Reply:
Yes, this just happened to be the bug from changing our size monitors into a separate report.